### PR TITLE
Release v1.2.5

### PR DESCRIPTION
**Pull request by jathanism**
Request to merge `develop` into `main` for release v1.2.5.

### CODE REVIEW

**Comment by jathanism** [comment]
Issues we should get merged in before releasing:

* [ ] Installed locked dependencies in final Docker image as well as dev](https://github.com/nautobot/nautobot/pull/1323)

File: nautobot/docs/release-notes/version-1.2.md [hunk 1]
```diff
@@ -140,6 +140,21 @@ All models that have `slug` fields now use `AutoSlugField` from the `django-exte
 
 Just as with the UI, the `slug` can still always be explicitly set if desired.
 
+## v1.2.5 (2022-??-??)
```
Comment (glennmatthews) [inline nautobot/docs/release-notes/version-1.2.md:143]:
```suggestion
## v1.2.5 (2022-02-02)
```

File: pyproject.toml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 name = "nautobot"
 # Primary package version gets set here. This is used for publishing, and once
 # installed, `nautobot.__version__` will have this version number.
-version = "1.2.4"
+version = "1.2.5-beta.1"
```
Comment (glennmatthews) [inline pyproject.toml:5]:
```suggestion
version = "1.2.5"
```

**Review by glennmatthews — APPROVED** [review]

**Review by jathanism — COMMENTED** [review]
Bump version and update release notes.

**Review by bryanculver — APPROVED** [review]
Looks fit to print!

:shipit:

**Pull request merged**